docs/traceflow-guide.md

```diff
@@ -0,0 +1,81 @@
+# Traceflow User Guide
+
+Antrea supports using Traceflow for network diagnosis.
+Creating a new Traceflow CRD triggers Traceflow module to inject packet into OVS, provide various observation points
+along the packet's path and populate these observations into status of Traceflow CRD.
+Users can start a new trace simply from either Kubectl or Antrea-Octant-Plugin and view Traceflow result via CRD or UI graph.
+We will also provide corresponding Antctl to start a new trace in the near future.
+
+For antrea-octant-plugin installation, please refer to [antrea-octant-installation](/docs/octant-plugin-installation.md).
+
+## Table of Contents
+
+- [Start a New Trace](#Start-a-New-Trace)
+  - [Using kubectl and YAML file](#using-kubectl-and-YAML-file)
+  - [Using Octant with antrea-octant-plugin](#Using-Octant-with-antrea-octant-plugin)
+- [View Traceflow Result and Graph](#View-Traceflow-Result-and-Graph)
+- [View Traceflow CRDs](#View-Traceflow-CRDs)
+
+## Start a New Trace
+You can choose to use either Kubectl together with YAML file or Octant UI to start a new trace.
+
+### Using kubectl and YAML file
+You can start a new trace by creating Traceflow CRD via Kubectl and a YAML file which contains the essential
+configuration of Traceflow CRD. An example YAML file of Traceflow CRD might look like this:
+```yaml
+apiVersion: ops.antrea.tanzu.vmware.com/v1alpha1
+kind: Traceflow
+metadata:
+  name: tf-test
+spec:
+  source:
+    namespace: default
+    pod: tcp-sts-0
+  destination:
+    namespace: default
+    pod: tcp-sts-2
+  packet:
+    ipHeader:
+      protocol: 6
+    transportHeader:
+      tcp:
+        srcPort: 10000
+        dstPort: 80
+```
+
```

Comment:
it would also be nice to show how to interpret the output when using yamls

Reply:
Got it, will add this part.

Reply:
Done.